modules/wallet/outputs.go

```diff
@@ -90,7 +93,8 @@ func (w *Wallet) Balance(full bool) (total consensus.Currency) {
 			if !full && knownOutput.age > w.age-AgeDelay {
 				continue
 			}
-			total += knownOutput.output.Value
+			// TODO: ???
+			total.Add(knownOutput.output.Value)
```

Comment:
hmm. Is it possible to move the overflow check to wherever we're adding outputs to the wallet? I feel like if adding an output causes an overflow, it should just be rejected. Having `Balance` return an error just seems backwards.